README.md

```diff
@@ -22,12 +22,10 @@ featurebrowser:
 ## Usage
 Run the featurebrowser generator from the command line:
 ```
-php bin/featurebrowser generate
+php bin\featurebrowser generate
```

Comment:
Actually no need if you do not want to use `--dev` (discussed above). I will close the PR.